domain-management/src/main/java/org/jboss/as/domain/management/security/KeystoreAttributes.java

```diff
@@ -71,6 +71,10 @@
             .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
             .build();
 
+    public static final SimpleAttributeDefinition GENERATE_SELF_SIGNED_CERTIFICATE_HOST = new SimpleAttributeDefinitionBuilder(org.jboss.as.controller.descriptions.ModelDescriptionConstants.GENERATE_SELF_SIGNED_CERTIFICATE_HOST, ModelType.STRING, true)
+            .setAllowExpression(true)
+            .setAllowNull(true)
```

Comment:
The others in here have .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES) and this one should too as it has the same handling.

Overall we're not good about adding this but since this particular class happens to be good about it.... :)